woocommerce/woocommerce-android
# [3/3] `Location` in `Address`: Shipping Labels adjustments

Related to: #5634


Other parts:
- https://github.com/woocommerce/woocommerce-android/pull/5644
- https://github.com/woocommerce/woocommerce-android/pull/5645

### Description

This PR adjusts app in areas of Shiping Labels.

### Testing instructions


Please test app in areas:
- **Shipping Labels**: when creating a label edit address. Use different locations and states, make sure all views updates correctly.
- **Order Editing**: open some order, click on edit of either shipping or billing details. Perform smoke tests, check if locations and states work correctly.
- **Order Creation**: this PRs break Order Creation for adding Customer! This is because supporting this in set of those PRs is I think out of scope. In very next PR I'll prepare fix with improved logic for this section.
